docs/Language_Naming_Format.rst

```diff
@@ -0,0 +1,9 @@
+.. important::
+
+    - `For language naming always consider the format that is used in the`
+      `corresponding Wikipedia page, (exception: use CPP instead`
+      `of cpp or c++);`
+    - `If the definition does not exist already a language definition`
+      `should be added while writing new bear;`
```

Comment:
`writing a`?

Reply:
@Utkarsh1308  I have done the requested changes

Reply:
I think you tagged the wrong @utkarsh2102 :)

Reply:
@utkarsh2102 Please review